### ISSUE TITLE
polars read_delta outputs too much information

### ISSUE THREAD

**Issue by 0xdarkman** [pola-rs/polars#14322]
### Checks

- [X] I have checked that this issue has not already been reported.
- [X] I have confirmed this bug exists on the [latest version](https://pypi.org/project/polars/) of Polars.

### Reproducible example

pl.read_delta(table_path, storage_options=storage_options)

Why am I seeing all the information below when reading delta table?

partition_values: {"name": String("foo")}
path: "name=foo/part-00000-aea5a35e-8164-4b72-b017-c85fe191c78c.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-91e85e96-1ec6-479f-ac6b-35a4d9edc058.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-b34142b8-ac55-4b08-be50-63cf803f2df6.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-cbc8c35d-b179-4449-b8e3-d812dc5c75c6.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-528c7682-3aec-4061-9a1b-b8e0b52568fb.c000.snappy.parquet"

### Log output

```shell
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-aea5a35e-8164-4b72-b017-c85fe191c78c.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-91e85e96-1ec6-479f-ac6b-35a4d9edc058.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-b34142b8-ac55-4b08-be50-63cf803f2df6.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-cbc8c35d-b179-4449-b8e3-d812dc5c75c6.c000.snappy.parquet"
partition_values: {"name": String("foo")}
path: "name=foo/part-00000-528c7682-3aec-4061-9a1b-b8e0b52568fb.c000.snappy.parquet"
```

### Issue description

I am expecting NOT to see all the partition_values and path.

### Expected behavior

To print output DF only.

### Installed versions

0.19.9

**Comment by stinodego** [comment]
I agree we shouldn't be showing this output. However, this is done by the `deltalake` package. Could you open an issue with them?
https://github.com/delta-io/delta-rs/issues

**Comment by 0xdarkman** [comment]
https://github.com/delta-io/delta-rs/issues/2176

I have noted this problem in deltalake 0.15.2 and not in deltalake 0.15.1

**Comment by ion-elgreco** [comment]
@stinodego this was fixed in the release 0.15.3 yesterday

**Comment by 0xdarkman** [comment]
yes, it works OK with 0.15.3
